packages/fxa-auth-server/lib/routes/devices-and-sessions.js

```diff
@@ -274,13 +274,12 @@ module.exports = (
           throw new error.featureNotEnabled();
         }
 
-        const [, device] = await Promise.all([
-          customs.checkAuthenticated(request, uid, 'invokeDeviceCommand'),
-          db.device(uid, target),
-        ]);
+        await customs.checkAuthenticated(request, uid, 'invokeDeviceCommand');
```

Comment:
This removal of `Promise.all` isn't necessary, I just think that running these two in parallel is not worth the tradeoff in code readability and it annoyed me while I was in here...